### DISCUSSION TITLE
How to connect and run kusto queries against ApplicationInsights and/or LogAnalytics?

### DISCUSSION THREAD

**Discussion by nulltoken** [dotnet/interactive#2862] [General]
Hello, 

From this [post](https://devblogs.microsoft.com/dotnet/polyglot-notebooks-december-2022-release/#improvements-to-sql-and-kql-with-the-new-dynamic-kernel-picker), one can see how to connect to a demo kusto cluster. However, I haven't been able to find how to properly use the `connect` command to query an Azure hosted Application Insights instance.

Any help would be greatly appreciated.

**Comment by Expecho** [comment]
There are 2 open issues that are asking for this feature, see [this](https://github.com/dotnet/interactive/issues/2849) and [this](https://github.com/dotnet/interactive/issues/1800). So far there does not seem to be much activity.
